src/Microsoft.ML.AutoML/AutoMLExperiment/IMetricManager.cs

```diff
@@ -12,13 +12,13 @@ namespace Microsoft.ML.AutoML
     public interface IMetricManager
     {
         bool IsMaximize { get; }
+
+        string MetricName { get; }
```

Comment:
>string [](http://example.com/codeflow?start=8&length=6)

I assume there is no app compat concern extending this interface. Extending interfaces on .NET Framework is a breaking change.